config/config.go

```diff
@@ -36,10 +53,12 @@ func NewViper() *viper.Viper {
 
 	v.MustBindEnv("kubectl", "KUBECTL_COMMAND")
 	v.MustBindEnv("objfreshthreshold", "KUBECOLOR_OBJ_FRESH")
+	v.MustBindEnv("pager", "KUBECOLOR_PAGER") // Don't bind PAGER yet as it should be overwritten by the config file
 
 	v.SetDefault("kubectl", "kubectl")
 	// mapstructure doesn't like "type X string" values, so we have to convert it via string(...)
 	v.SetDefault(PresetKey, string(PresetDefault))
+	v.SetDefault("paging", "always")
```

Comment:
nitpick: use the const you introduced


```suggestion
	v.SetDefault("paging", string(PagingAlways))
```